pkg/virt-controller/watch/application.go

```diff
@@ -310,6 +318,13 @@ func (vca *VirtControllerApp) configModificationCallback() {
 	}
 }
 
+// Update virt-controller log verbosity on relevant config changes
+func (vca *VirtControllerApp) shouldChangeLogVerbosity() {
+	verbosity := vca.clusterConfig.GetVirtHandlerVerbosity(vca.host)
+	log.Log.SetVerbosityLevel(int(verbosity))
+	log.Log.V(4).Infof("set verbosity to %d", verbosity)
```

Comment:
would it make sense to set the verbosity for these log messages indicating what the verbosity is to something high, so we can always view something that indicates when the verbosity changes?

Reply:
yup, done